packages/enzyme-adapter-react-16/src/ReactSixteenAdapter.js

```diff
@@ -588,6 +609,22 @@ class ReactSixteenAdapter extends EnzymeAdapter {
       return wrappedComponent;
     };
 
+    const renderElement = (elConfig, ...rest) => {
+      const renderedEl = renderer.render(elConfig, ...rest);
+
+      const typeIsExisted = !!(renderedEl && renderedEl.type);
```

Comment:
when would this not be true?

Reply:
Yep, sometimes `type` in Element is `undefined ` with `$$typeof` is `(react.element)`. I'm not sure I know which cases it can be `undefined`.

Reply:
We've got a bunch of existing helpers for this kind of thing; is there one of those we could use here?

Reply:
I don't see any helper to check is existed type in element. Do you want to me create a helper to use it?